index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="css/style.css">
+        <title></title>
+        <header>
+        
+        </header>
+    </head>
+    <body>
+        <div class="heading">
+            <h1>Dr. Norman Borlaug</h1>
+            <h5>The man who saved a billion lives</h5>
+        </div>
```

Comment:
I think This Section of the Page should be enclosed in a `Section` tag, and also line **15** should be enclosed in a `header` tag as it is an important headline, This is important for the semantics rule to be accomplished.

Reply:
- [ ] Kindly always make sure you provide the `why` the changes were requested. You may also consider directing the code author to some online documentation to get to know the reason why the changes were required.
`Hint: You can check` [here](https://www.w3schools.com/tags/tag_header.asp) `when to user the header tag because I can see where you tell to put it in your code review is not the right place for <header> to be placed at`